### PR TITLE
Simplify Getting Started

### PR DESCRIPTION
**Pull request by bricelam**
Part of #481

### CODE REVIEW

File: entity-framework/core/get-started/index.md [hunk 1]
```diff
@@ -1,33 +1,164 @@
...
+
+---
+
+## Create the model
+
+Define a context class and entity classes that make up the model.
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+* In the **Models** folder create **Model.cs** with the following code
```
Comment (smitpatel) [inline entity-framework/core/get-started/index.md]: Where did Models folder come from?

File: entity-framework/core/get-started/index.md [hunk 2]
```diff
@@ -1,33 +1,164 @@
...
+## Create the database
+
+The following steps use [migrations](xref:core/managing-schemas/migrations/index) to create a database.
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+* Run the following commands:
+
+  ``` Console
+  dotnet tool install --global dotnet-ef --version 3.0.0-*
```
Comment (smitpatel) [inline entity-framework/core/get-started/index.md]: Will it match 3.0.0-* to 3.0.0 once we RTM?

File: entity-framework/core/get-started/index.md (hunk 1 above)
Comment (bricelam) [reply entity-framework/core/get-started/index.md]: Oops, left over from the ASP.NET walkthrough I adapted

File: entity-framework/core/get-started/index.md (hunk 2 above)
Comment (bricelam) [reply entity-framework/core/get-started/index.md]: Will update on the day we publish as part of #1588

**Pull request merged**